docs/user-guide/building-assistants.rst

```diff
@@ -548,15 +587,31 @@ The intents and entities will need to be added to your domain as well:
      - use_case
 
 A story for a form is very simple, as all the slot collection form happens inside the form, and
-therefore doesn’t need to be covered in your stories.
+therefore doesn’t need to be covered in your stories. You just need to write stories showing when the form should be activated. For the sales form, add this story
+to your ``stories.md`` file:
+
 
 .. code-block:: md
 
    ## sales form
    * contact_sales
-       - sales_form
-       - form{"name": "sales_form"}
-       - form{"name": null}
+       - sales_form                   <!--Run the sales_form action-->
+       - form{"name": "sales_form"}   <!--Activate the form-->
+       - form{"name": null}           <!--Deactivate the form-->
+
+
+.. cssclass:: longtable
+
++----------------------------------+--------------------------------------+
+|.. code-block:: md                |.. code-block:: md                    |
+|                                  |                                      |
+|   ## sales form                  |     <!--What's happening here-->     |
+|      - sales_form                |     <!--Run the sales_form action--> |
+|      - form{"name": "sales_form"}|     <!--Activate the form-->         |
+|      - form{"name": null}        |     <!--Deactivate the form-->       |
++----------------------------------+--------------------------------------+
```

Comment:
why do we have both? I feel like just the first one is enough?

Reply:
that was an experiment I forgot to remove